### PR TITLE
[Draft] Adding workflow for container push

### PR DESCRIPTION
**Pull request by Yarboa**
Resolve Issue #140
*Note* It will not run on each PR, test workflow

Using podman registry
https://github.com/redhat-actions/podman-login#readme

Signed-off-by: Yariv Rachmani <yrachman@redhat.com>

### CODE REVIEW

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Building Base Image
+
+on:
+  pull_request:
```
Comment (mwperina) [inline .github/workflows/image.yml:4]: Do you really want to build an image on every update of each PR? Shouldn't the image be built only after merged to main branch?

**Review by mwperina — COMMENTED** [review]
I still don't understand why do we need to push images of unstable build into public repository, what's wrong to create them before running the test suite?

File: .github/workflows/image.yml (hunk 1 above)
Comment (engelmi) [reply .github/workflows/image.yml:4]: I think this is just for testing if we can push images to ghcr with this approach.

**Comment by Yarboa** [comment]
> I still don't understand why do we need to push images of unstable build into public repository, what's wrong to create them before running the test suite?

It is not wrong, it just take time to build it, 
Once there is prepared build, you pull 987M and run the binaries for build
This image will change once new rpm/update needed.

Think of running the image also on aarch 64 or arm,

**Review by Yarboa — COMMENTED** [review]
@engelmi 

Failures related to the project configuration
docker push ghcr.io/containers/hirte/hirte-builder:1.0.0
docker push ghcr.io/containers/hirte-builder:1.0.0

While private repo is working
docker push ghcr.io/yarboa/hirte-builder:1.0.0

Please refer this 
https://github.com/Yarboa?tab=packages

Definitely needed to add permission of packages for https://github.com/orgs/containers/packages

**Comment by Yarboa** [comment]
Currently found w/a we can continue discussion in #140

**Pull request closed without merging**